README.md

```diff
@@ -1 +1,136 @@
-# Hello__Microverse-
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```

Comment:
- [ ] Your sections are missing in your Repo, Please add the sections that you removed and please make sure that your table should work according to that sections. Except for the `FAQ` all the sections are required in order to make your documentation looks more professional.

### Professional README

![image](https://github.com/CVILLA09/hello-microverse/assets/66476329/2463f41a-66e9-4458-b2e3-c94cc430fda5)

Reply:
@skyv26 Hello , I delete those sections because my project does not have live demo , Usage , Deployment .... This is a simple project and has one line code kindly tell me what should I do ?

Reply:
@Nelofarzabi please keep remembering according to the professional template except `FAQ` rest of everything is required. So kindly add the required section to make your `README` looks professional.